### DISCUSSION TITLE
[WASM] Custom index.html no longer working

### DISCUSSION THREAD

**Discussion by DierkDroth** [unoplatform/uno#8384] [Q&A]
### Current behavior

I followed the instructions [here ](https://github.com/unoplatform/Uno.Wasm.Bootstrap#indexhtml-content-override)to create a custom index.html (attached).

Please 
- edit CustomIndex.Wasm.csproj
- search for !!! and take action as advised
- note that CustomIndex WASM no longer starts but shows errors attached in browser

[CustomIndex.zip](https://github.com/unoplatform/uno/files/8308863/CustomIndex.zip)
[localhost-1647680152131.zip](https://github.com/unoplatform/uno/files/8308864/localhost-1647680152131.zip)

**Comment by jeromelaban** [comment]
Thanks for the report. This is a documentation change as the current master is running on a different version of .NET.

You can find the appropriate documentation on the 3.3 branch: https://github.com/unoplatform/Uno.Wasm.Bootstrap/tree/release/stable/3.3

**Reply by DierkDroth** [reply]
Thanks @jeromelaban. That got me going...
